styles/variables-silicon.less

```diff
@@ -158,34 +158,45 @@
 @agate-input-decorator-focus-border-width: 0;
 @agate-input-decorator-height: 60px;
 @agate-input-focused-opacity: 1;
-@agate-input-font-size: 28px;
-@agate-input-height: 33px;
-@agate-input-opacity: 0.5;
-@agate-input-icon-display: flex;
-@agate-input-icon-align: center;
-@agate-input-padding-top-bottom: 21px;
-@agate-input-padding-right: 33px;
-@agate-input-padding-left: 0;
-@agate-input-icon-after-padding-top-bottom: 21px;
-@agate-input-icon-after-padding-right: 21px;
+@agate-input-font-size: 30px;
+@agate-input-hasiconbefore-padding-left: 6px;
+@agate-input-height: 54px;
 @agate-input-icon-after-padding-left: 9px;
-@agate-input-icon-before-padding-top-bottom: 21px;
-@agate-input-icon-before-padding-right: 9px;
+@agate-input-icon-after-padding-right: 21px;
+@agate-input-icon-after-padding-top-bottom: 27px;
+@agate-input-icon-align: center;
 @agate-input-icon-before-padding-left: 21px;
-@agate-input-text-align: left;
+@agate-input-icon-before-padding-right: 0;
+@agate-input-icon-before-padding-top-bottom: 21px;
+@agate-input-icon-display: flex;
+@agate-input-large-icon-after-padding-bottom: 30px;
+@agate-input-large-icon-after-padding-left: 15px;
+@agate-input-large-icon-after-padding-top: 27px;
+@agate-input-large-icon-before-padding-bottom: @agate-input-large-icon-after-padding-bottom;
+@agate-input-large-icon-before-padding-right: 6px;
+@agate-input-large-icon-before-padding-top: @agate-input-large-icon-after-padding-top;
+@agate-input-large-icon-font-size: 81px;
+@agate-input-large-icon-height: 39px;
+@agate-input-max-width: 930px;
+@agate-input-min-width: 360px;
+@agate-input-opacity: 0.5;
```

Comment:
The opacity should be 0.3(Normal state) if the placeholder is displayed and 0.7(Entered state) if there is a text, according to UI spec

Reply:
You should check again as I changed the implementation, but for some there is now a combination of opacities, one on the parent and then on he specific element.